### ISSUE TITLE
💡 [FEATURE] - Web Dev Project

### ISSUE THREAD

**Issue by Abhay182005dat** [jfmartinz/ResourceHub#523]
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

Greetings  Sir !!

Sir I would like to make a Netflix clone as a project as add it as a resource under Web Dev project as i am learning Web Dev right now and would help me in learning Web Dev by contributing here.

So could you please assign it to me if you think my request is good. :)

Regards.

### Explain your solution

I hope my request i spretty straight forward. :)

### Any alternative approaches/features

We can add similar type of Projects in the web Dev category which will help many.

**Comment by jfmartinz** [comment]
Yes, you can but give the github URL i think for source code, thanks

**Comment by Abhay182005dat** [comment]
Yes sir i will give(You mean the project url in the github right?) 

Thank you for accepting my request  :)

**Comment by jfmartinz** [comment]
yes
